resdk/resources/base.py

```diff
@@ -105,4 +102,6 @@ def update(self):
         self._update_fields(self.api(self.id).get())
 
     def __repr__(self):
-        return u"{} <id: {}, slug: '{}', name: '{}'>".format(self.__class__.__name__, self.id, self.slug, self.name)
+        """Format resource name."""
+        return u"{} <id: {}, slug: '{}', name: '{}'>".format(self.__class__.__name__,
```

Comment:
Drop the `u` from `u"{}..."` since all strings will automatically be unicode due to `from __future__ import unicode_literals`.